src/clients/java/docs.zig

```diff
@@ -145,7 +145,12 @@ pub const JavaDocs = Docs{
     \\CreateAccountResultBatch accountErrors = client.createAccounts(accounts);
     ,
 
-    .create_accounts_documentation = "",
+        .create_accounts_documentation =
+            \\The 128-bit fields like `id` and `user_data` have a few
+            \\overrides to make it easier to integrate. You can either
+            \\pass in a long, a pair of longs (least and most
+            \\significant bits), or a `UInt128`.
```

Comment:
```suggestion
            \\significant bits), or a `byte[]`.
```
The overloaded setter accepts an array of 16 bytes `byte[]`, likewise, all getters can return an array too.

`UInt128` isn't a class, it's a helper with static methods and enums for converting 128-bit little-endian unsigned integers between the Java's types `long`, `UUID`, `BigInteger` and `byte[]`.

For example:
```java
// Using long
account.setId(137);
long id = account.getId(UInt128.LeastSignificant);
assertEquals(137L, id);
assertEquals(0L, account.getId(UInt128.MostSignificant));
```

```java
// Using bytes
byte[] bytes = new byte[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6 };
account.setId(bytes);
byte[] id = account.getId();
assertArrayEquals(bytes, id);
```

Reply:
Edited with your notes, thank you!